lib/src/main/java/com/auth0/msg/AbstractMessage.java

```diff
@@ -0,0 +1,208 @@
+package com.auth0.msg;
```

Comment:
You haven't refactored for package name?

Reply:
TODO